jenkins/run-pytests

```diff
@@ -25,3 +25,9 @@ if [[ ${pytest_sts} -ne 0 ]]; then
 fi
 final_sts=$(( black_sts + flake_sts + pytest_sts))
 exit ${final_sts}
+printf -- "\nPython coverage ...\n"
+pytest --cov=pbench
```

Comment:
Why not add `--cov=pbench` to line 21 where we specify an `coverage.html` file to be created so as not to pollute the logs.  Then we can modify Jenkins to capture the `coverage.html` file as an artifact of the build.